LimitSetting/python/winoElectroweakLimits.py

```diff
@@ -18,12 +18,14 @@
              '200', '300', '400', '500', '600', '700', '800', '900', '1000',
              '2000', '3000', '4000', '5000', '6000', '7000', '8000', '9000', '10000']
 
-if arguments.era.startswith("2017") or arguments.era.startswith("2018") or arguments.era == "all20178" or arguments.era == "run2":
+if arguments.era.startswith("2017") or arguments.era.startswith("2018") or arguments.era.startswith("2022") or arguments.era.startswith("2023") or arguments.era == "all20178" or arguments.era == "run2" or arguments.era == "run2run3" or arguments.era == "run3":
 	masses.extend(['1000', '1100'])
 	lifetimes = ['0.2', '0.3', '0.4', '0.5', '0.6', '0.7', '0.8', '0.9', '1'] + lifetimes
+if arguments.era.startswith("2022") or arguments.era.startswith("2023") or arguments.era == "run2run3" or arguments.era == "run3":
+	masses.extend(['1200'])
 
 # define maximal set of masses/lifetimes for datacard combinations
-allMasses = masses + ['1000', '1100']
+allMasses = masses + ['1000', '1100', '1200']
```

Comment:
since this is only true for 2022 on can you just add an if statement for 2022/2023? I hate that our code is like that but may as well be consistent

Reply:
@carriganm95 just did it here https://github.com/OSU-CMS/DisappTrks/pull/51/commits/793627ee5605c5b8d563a23fb81471c1bf21015a. Is it better?

Reply:
perfect, approved